MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h

```diff
@@ -47,7 +47,7 @@ typedef NS_ENUM(NSUInteger, MUXSDKViewOrientation) {
     AVPlayer *_player;
     AVPlayerItem *_playerItem;
     id _timeObserver;
-    MUXSDKPlayerState _state;
+    volatile MUXSDKPlayerState _state;
```

Comment:
It's really more of a precaution cause when I was debugging I noticed this variable is constantly set like the breakpoint for setting its value kept pausing in a loop. And I noticed some weird cases where the check for the state was going into the incorrect block of code cause when it checked the conditional state had different value than when performing the logic inside the block. Once I added volatile I didn't see this behavior anymore because it avoids the compilers optimizations that could cause us to get the previous value instead.

Reply:
😬  got it